packages/hooks/src/createAddHook.js

```diff
@@ -70,7 +74,7 @@ function createAddHook( hooks ) {
 			// we're adding would come after the current callback, there's no
 			// problem; otherwise we need to increase the execution index of
 			// any other runs by 1 to account for the added element.
-			( hooks.__current || [] ).forEach( ( hookInfo ) => {
+			hooks.__current.forEach( ( hookInfo ) => {
```

Comment:
Why this change?

Reply:
Thank you for highlighting. I didn't investigate why this included the fallback but the types suggest it _is_ safe. Not that `createAddHook( hooks )` accepts a parameter of type `Hooks`:

https://github.com/WordPress/gutenberg/blob/77f872a1a50f39b8f618db147ac8a54bb3fef63e/packages/hooks/src/createAddHook.js#L19-L26

Which is an object with a `__current` property of type `Current[]`:

https://github.com/WordPress/gutenberg/blob/77f872a1a50f39b8f618db147ac8a54bb3fef63e/packages/hooks/src/index.js#L22-L30

We can also see that this package exposes `createHooks` and the default hooks:

https://github.com/WordPress/gutenberg/blob/77f872a1a50f39b8f618db147ac8a54bb3fef63e/packages/hooks/src/index.js#L53-L73

Inside `createHooks` —the only place these individual hook creators can be called as far as I can tell— `__current` does seem to be defined:

https://github.com/WordPress/gutenberg/blob/77f872a1a50f39b8f618db147ac8a54bb3fef63e/packages/hooks/src/createHooks.js#L15-L24

I do believe it's safe to remove the fallback.

Reply:
Right on, thanks for digging into it.